Kernel/Arch/aarch64/PageDirectory.h

```diff
@@ -171,4 +177,51 @@ class PageDirectoryPointerTable {
     u64 raw[512];
 };
 
+class PageDirectory final : public AtomicRefCounted<PageDirectory> {
+    friend class MemoryManager;
+
+public:
+    static ErrorOr<NonnullLockRefPtr<PageDirectory>> try_create_for_userspace();
+    static NonnullLockRefPtr<PageDirectory> must_create_kernel_page_directory();
+    static LockRefPtr<PageDirectory> find_current();
+
+    ~PageDirectory();
+
+    void allocate_kernel_directory();
+
+    FlatPtr ttbr0() const
```

Comment:
(the body seems to be more descriptive than the method name… 😅)

Reply:
Yes you are correct again, but I just used the naming scheme that the existing x86_64 code also used, which was to use the name of the register that this value will be stored in. Could change it, but I think it's fine and it can always be improved later on :)